setup.py

```diff
@@ -54,7 +54,7 @@ def optional_requirements(
 ## Package Setup ##
 ###################
 setup(
-    name="ethyca-fides",
+    name="ethyca_fides",
```

Comment:
do we know if this change will affect anything in how fides is installed or imported?

Reply:
@andres-torres-marroquin one thing to test specifically may be that fidesplus can build with the package that's published from this branch

Reply:
@andres-torres-marroquin one thing to test specifically may be that fidesplus can build with the package that's published from this branch